bin/java

```diff
@@ -165,17 +168,20 @@ install_java_with_overlay() {
 	cp -r "${JVM_COMMON_BUILDPACK_DIR}/opt/tools/"* "${build_dir}/.heroku"
 
 	# Write export script for subsequent buildpacks to ensure they can use the installed OpenJDK.
+	# Note that this is not using JVM_COMMON_BUILDPACK_DIR but the directory of the buildpack that
+	# invoked this function to write it to the correct location.
+	#
 	# See: https://devcenter.heroku.com/articles/buildpack-api#composing-multiple-buildpacks
-	cat <<-EOF >"${JVM_COMMON_BUILDPACK_DIR}/export"
+	cat <<-EOF >>"${host_buildpack_dir}/export"
```

Comment:
Switching to append here as the host buildpack might have already written an export script.